src/encoded/static/components/matrix.js

```diff
@@ -197,7 +197,10 @@ class Matrix extends React.Component {
                                 <div className="row">
                                     <div className="col-sm-11">
                                         <div>
-                                            <h1>{context.title}</h1>
+                                            <h1>
+                                                <div className="matrix-title">{context.title}</div>
```

Comment:
You’ll need to break this up somehow. `<h1>` elements [can only contain phrasing elements](https://w3c.github.io/html-reference/h1.html#h1) and a div [isn’t one](https://w3c.github.io/html-reference/common-models.html#common.elem.phrasing).